release/preview/nanoserver2022/meta.json

```diff
@@ -0,0 +1,24 @@
+{
+    "IsLinux": false,
+    "PackageFormat": "PowerShell-${PS_VERSION}-win-x64.zip",
+    "osVersion": "Nano Server, LTSC 2022",
+    "shortTags": [
+        {
+            "Tag": "ltsc2022"
+        }
+    ],
+    "tagTemplates": [
+        "#psversion#-nanoserver-#shorttag#",
+        "#psversion#-nanoserver-#shorttag#-#fulltag#",
+        "preview-nanoserver-#shorttag#"
+    ],
+    "manifestLists": [
+        "nanoserver"
+    ],
+    "Base64EncodePackageUrl": true,
+    "TestProperties": {
+        "size": 1
+    },
+    "UseAcr": true,
```

Comment:
Does ACR support these newer images yet?

Reply:
ACR has been updated to test 2022, but will be marked internal (via `isPrivate`)

Reply:
The public ACR doesn't support this yet... We have access to a VERY private prerelease that does.